benchmarks/lockhammer/src/lockhammer.c

```diff
@@ -254,19 +263,24 @@ void* hmr(void *ptr)
            cores to become ready and starts all cores with a write to the
            shared memory location */
 
-        /* Set affinity to core 0 */
-        CPU_SET(0, &affin_mask);
-        sched_setaffinity(0, sizeof(cpu_set_t), &affin_mask);
+        if (!safemode) {
```

Comment:
We probably need to have sched_setaffinity even in safemode. Because sched_setaffinity doesn't need root permission.

Reply:
Agreed, we can keep sched_setaffinity(), we really only need the SCHED_FIFO turned off.

Reply:
Initially I removed this because I was worried about delayed sync start times with affinity set but with FIFO disabled that shouldn't be as much of a problem.  This should also improve accuracy slightly.